xbmc/guilib/AspectRatio.h

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright (C) 2005-2024 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include <cstdint>
+
+// image alignment for <aspect>keep</aspect>, <aspect>scale</aspect> or <aspect>center</aspect>
+#define ASPECT_ALIGN_CENTER 0
+#define ASPECT_ALIGN_LEFT 1
+#define ASPECT_ALIGN_RIGHT 2
+#define ASPECT_ALIGNY_CENTER 0
+#define ASPECT_ALIGNY_TOP 4
+#define ASPECT_ALIGNY_BOTTOM 8
+#define ASPECT_ALIGN_MASK 3
+#define ASPECT_ALIGNY_MASK ~3
```

Comment:
Lets make all theses `static constexpr unsigned int`.

Reply:
isn't static not needed for constexpr variables in a header?

Reply:
Ah yes, you are correct. `inline constexpr` would probably be the best if I my understanding is correct but we haven't done that anywhere and our code guidelines say just `constexpr`as well.

Reply:
Made them `inline constexpr`.